polyglot-scala/src/main/scala/org/sonatype/maven/polyglot/scala/ScalaMapping.scala

```diff
@@ -1,9 +1,8 @@
 /**
- * Copyright (c) 2012 to original author or authors
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
+ * Copyright (c) 2012 to original author or authors All rights reserved. This
```

Comment:
This looks a bit ugly and is unexpected (the merged lines). Maybe, because it was formatted without any config before? Not a showstopper, but maybe you could retry with the original files?

Reply:
Seems you were right, re-reformatted with master checkout first and looks better

Reply:
Very nice. 👏